metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
The problem is, that response times are sometimes lower then 1 millisecond, like 0.044 ms, means we would need a float. This could actually also be an issue in packetbeat I would assume?